### DISCUSSION TITLE
Workflow for an issue that needs submodule and supermodule changes

### DISCUSSION THREAD

**Discussion by jhowlett-scottlogic** [xunit/xunit#2558] [Question] [answered]
I was wondering what the recommended workflow is for committing back to a sub and super module. I have the behavioural changes in a sub module (xunit/assert.xunit) but the testing is in the super module (xunit/xunit). 

do i make a PR on the sub module first and then a PR on the super module?

**Comment by bradwilson** [comment, marked as answer]
I do it as two PRs.

That said, I sometimes see people successfully send up a single PR that has both things in it and _I have no idea how it's done_. 😭😂
